### PR TITLE
Proposal for contribution guide and bug request

### PR DESCRIPTION
**Pull request by jannaahs**
I found a good text for the bug reports and used it.
I would like to have a short contribution guide.

Both are in german and english, because there are a lot of german speakers and we agreed to change the documentation to english #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+- Pull Request reviewen lassen
+
+
+**english**
+
+- create a fork
+- commit changes to the fork
+- create a merge request
+- let the pull request be reviewed
+
```
Comment (andygrunwald) [inline README.md]: An alternative would be to link here one of the good guides like
- https://gist.github.com/Chaser324/ce0505fbed06b947d962
or
- https://help.github.com/articles/creating-a-pull-request-from-a-fork/

File: README.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+## Bug Report
+
+**deutsch**
+
+Falls du eine Issue anlegst sollte dieses einen klaren Titel mit einer genauen Beschreibung des enthalten. Du solltest so viele relevante Informationen wie möglich angeben und auch ein Code Beispiel, welches den Fehler demonstriert. Ein Bug Report soll dir und anderen helfen  den Fehler zu reproduzieren und einen Fix zu entwickeln.
```
Comment (andygrunwald) [inline README.md]: > Falls du eine Issue anlegst sollte dieses einen klaren

"Falls du eine Issue anlegst, sollte ..."

> genauen Beschreibung des enthalten

"des" is not necessary here.

> anderen helfen  den Fehler zu reproduzieren

Two whitespaces between *helfen  den*

File: README.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+
+Bitte denk daran, dass ein Bug Report erstellt werden in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden. Erwarte nicht, dass der Bug Report automatisch als Aufgabe gesehen wird oder sofort gelöst wird. Das Erstellen eines Bug Reports soll dabei helfen dir und anderen den ursprung des Problems zu ermitteln.
```
Comment (andygrunwald) [inline README.md]: > Bitte denk daran, dass ein Bug Report erstellt werden in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden.

Bitte denk daran, dass Bug Reports erstellt werden, in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden.

> anderen den ursprung

"den Ursprung"

File: README.md [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+**english**
+
+- create a fork
+- commit changes to the fork
+- create a merge request
+- let the pull request be reviewed
+
+## Bug Report
+
+**deutsch**
```
Comment (andygrunwald) [inline README.md]: Uppercase? Because it is a headline

File: README.md [hunk 5]
```diff
@@ -1,3 +1,36 @@
...
+
+Falls du eine Issue anlegst sollte dieses einen klaren Titel mit einer genauen Beschreibung des enthalten. Du solltest so viele relevante Informationen wie möglich angeben und auch ein Code Beispiel, welches den Fehler demonstriert. Ein Bug Report soll dir und anderen helfen  den Fehler zu reproduzieren und einen Fix zu entwickeln.
+
+Bitte denk daran, dass ein Bug Report erstellt werden in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden. Erwarte nicht, dass der Bug Report automatisch als Aufgabe gesehen wird oder sofort gelöst wird. Das Erstellen eines Bug Reports soll dabei helfen dir und anderen den ursprung des Problems zu ermitteln.
+
+**english**
```
Comment (andygrunwald) [inline README.md]: Uppercase? Or lowercase? Because it is the first word in a headline

File: README.md (hunk 1 above)
Comment (jannaahs) [inline README.md]: Do you want to link one of those instead of the english version or both?

File: README.md (hunk 5 above)
Comment (jannaahs) [inline README.md]: Uppercase

**Comment by andygrunwald** [comment]
@jannaahs I've updated the PR a little bit. WHat i did:

- Added a `CONTRIBUTION.md`: If you create now an issue a yellow box will appear that recommends to read the contribution guide first
- Added a `CONTRIBUTION-DE.md`: The contribution guide in german.
- Added a Issue Template: [Github provides issue templates](https://github.com/blog/2111-issue-and-pull-request-templates). If someone creates an issue, he or she needs sees a kind of template to fill out instead of a complete free text

I hope you agree with those changes. If yes, feel free to merge them.

**Pull request merged**